sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-255.3.ebuild

```diff
@@ -572,7 +562,7 @@ multilib_src_install_all() {
 	# Flatcar: Use an empty preset file, because systemctl
 	# preset-all puts symlinks in /etc, not in /usr. We don't use
 	# /etc, because it is not autoupdated. We do the "preset" above.
-	rm "${ED}$(usex split-usr '' /usr)/lib/systemd/system-preset/90-systemd.preset" || die
+	# rm "${ED}$(usex split-usr '' /usr)/lib/systemd/system-preset/90-systemd.preset" || die
```

Comment:
Why is this removed? This could be the reason why some enablement symlinks end up in the  /etc upperdir

Reply:
I guess we could make use of presets if we add a step in the image postprocessing where we evaluate the presets to create symlinks from them.

Reply:
/lib/systemd/system-preset/90-systemd.preset does not exist anymore on this version of systemd.

Reply:
hmm, I think the path is incorrect due to the split-usr usex, need to check again.

Reply:
`rm: cannot remove '/var/tmp/portage/sys-apps/systemd-255.3/image/lib/systemd/system-preset/90-systemd.preset': No such file or directory` - this is the log message.

Reply:
`/var/tmp/portage/sys-apps/systemd-255.3/image/usr/lib/systemd/system-preset/90-systemd.preset` exists.

Reply:
@pothos testing now locally with

```
diff --git a/sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-255.3.ebuild b/sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-255.3.ebuild
index 7e8266d7d2..b1584034c7 100644
--- a/sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-255.3.ebuild
+++ b/sdk_container/src/third_party/coreos-overlay/sys-apps/systemd/systemd-255.3.ebuild
@@ -295,7 +295,7 @@ src_configure() {

 # Flatcar: Our function, we use it in some places below.
 get_rootprefix() {
-       usex split-usr "${EPREFIX:-/}" "${EPREFIX}/usr"
+       "${EPREFIX}/usr"
 }
 multilib_src_configure() {
        local myconf=(
@@ -562,8 +562,8 @@ multilib_src_install_all() {
        # Flatcar: Use an empty preset file, because systemctl
        # preset-all puts symlinks in /etc, not in /usr. We don't use
        # /etc, because it is not autoupdated. We do the "preset" above.
-       # rm "${ED}$(usex split-usr '' /usr)/lib/systemd/system-preset/90-systemd.preset" || die
-       insinto $(usex split-usr '' /usr)/lib/systemd/system-preset
+       rm "${ED}/usr/lib/systemd/system-preset/90-systemd.preset" || die
+       insinto /usr/lib/systemd/system-preset
        doins "${FILESDIR}"/99-default.preset

        # Flatcar: Do not ship distro-specific files (nsswitch.conf

```